docs/proposed/psa-driver-interface.md

```diff
@@ -3,1465 +3,8 @@ PSA Cryptoprocessor Driver Interface
 
```

Comment:
If we remove the substance of this document, we should stop linking to it in `README.md` and elsewhere in our code base.